test-resources/rest-service-specs/test-service.json

```diff
@@ -0,0 +1,209 @@
+{
```

Comment:
I do not see any generated entities from this definition that are then called in E2E to the backend you created.

Reply:
no it is there.

Reply:
This is because I wanted to do it on a separate PR. I was hoping for some feedback on the setup first. ;)

Reply:
ok.